scality/metalk8s
# Add a test framework

**What has been added**
- ~~Dummy test added~~ : useless now, removed
- Ability to tag tests based on ci or local
- Add a test to list pods

**How to test**
You can run a local tests session using `tox -e local-tests` this command will select all the tests with the `local` tag and run them

Fixes: #723 #716 #715 #714 

**Update**: this part should be enough got the mentioned issues, lets land this ASAP to continue the rest of the issue in separate PRs
